frontend/src/layout/navigation-3000/sidebars/personsAndGroups.ts

```diff
@@ -170,6 +169,14 @@ export const personsAndGroupsSidebarLogic = kea<personsAndGroupsSidebarLogicType
         // kea-typegen doesn't like selectors without deps, so searchTerm is just for appearances
         debounceSearch: [(s) => [s.searchTerm], () => true],
     })),
+    listeners(({ cache }) => ({
+        loadPersons: async ({ url }) => {
+            const offset = url ? parseInt(new URL(url).searchParams.get('offset') || '0') : 0
+            if (offset === 0) {
+                cache.requestedPersons.length = 0 // Clear cache
```

Comment:
🤔
```suggestion
                cache.requestedPersons = [] // Clear cache
```

Reply:
Just performance, `.length = 0` is the fastest way to do this (weirdly JS has no `.clear()`, but this does the same as `.clear()` in Python)